Source/SwiftLintFramework/Rules/RuleConfigurations/PrivateUnitTestConfiguration.swift

```diff
@@ -36,13 +36,12 @@ public struct PrivateUnitTestConfiguration: RuleConfiguration, Equatable {
     }
 
     public mutating func applyConfiguration(configuration: AnyObject) throws {
-        guard let configurationDict = configuration as? [String: AnyObject],
-            regexString = configurationDict["regex"] as? String else {
-                throw ConfigurationError.UnknownConfiguration
+        guard let configurationDict = configuration as? [String: AnyObject] else {
```

Comment:
It's [in there](https://github.com/cfilipov/SwiftLint/blob/5fd99cd1d77f0ef1f7df1481eb675d6a13ae3f84/CHANGELOG.md). Do you mean calling out the fix for the configuration part as well?